connectors/aws/aws-s3/src/main/java/io/camunda/connector/aws/s3/model/response/DownloadResponse.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.aws.s3.model.response;
+
+import io.camunda.document.Document;
+
+public record DownloadResponse(String bucket, String key, Document document, Object content) {}
```

Comment:
@mathias-vandaele Is it either a `Document` or the `content`? 
If so please create a simple sealed type which indicates that.